frontend/src/middlewares/socket/socket.middleware.ts

```diff
@@ -0,0 +1,32 @@
+import { Middleware, AnyAction } from '@reduxjs/toolkit';
+import { io } from 'socket.io-client';
+import { ENV, SocketEvent, DataStatus } from 'common/enums/enums';
+import { Comment } from 'common/types/types';
+import { ActionType as EpisodeActionType } from 'store/episode/common';
+import { episode as episodeAction } from 'store/actions';
+
+type Next = (action: AnyAction) => void;
+
+const socket = io(ENV.SOCKET_SERVER);
+
+const socketMiddleware: Middleware = ({ dispatch }) => (next): Next => {
```

Comment:
```suggestion
const socket = io(ENV.SOCKET_SERVER);

const socketMiddleware: Middleware = ({ dispatch }) => (next) => {
```
we really need it? 🤔

Reply:
Yep. Without this type checks failed